x/clp/alias.go

```diff
@@ -15,6 +15,8 @@ const (
 	MaxWbasis         = types.MaxWbasis
 	PoolThrehold      = types.PoolThrehold
 	PoolUnitsMinValue = types.PoolUnitsMinValue
+	SwapType          = types.SwapType
+	TxFeeMultiplier   = types.TxFeeMultiplier
 )
 
```

Comment:
Just a friendly reminder, that the SDK has moved away from the `alias.go` file convention -- it really didn't add any benefits. That being said, feel free to keep it if it makes sense for your team.